bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/dto/AVMFritzBaseModel.java

```diff
@@ -90,15 +99,32 @@
     @XmlElement(name = "powermeter")
     private PowerMeterModel powermeterModel;
 
+    @XmlElement(name = "colorcontrol")
+    private ColorControlModel colortrolModel;
```

Comment:
Spelling mistake? Be aware of more changes in this file.

```suggestion
    private ColorControlModel colorControlModel;
```